Sources/Core/GTMSessionFetcher.m

```diff
@@ -3995,6 +4019,21 @@ - (void)setUsingBackgroundSession:(BOOL)flag {
   }  // @synchronized(self)
 }
 
+- (BOOL)stopFetchingTriggersCompletionHandler {
+  return _stopFetchingTriggersCompletionHandler;
+}
+
+- (void)setStopFetchingTriggersCompletionHandler:(BOOL)flag {
+  @synchronized(self) {
```

Comment:
I think you can skip the sync, _initialBeginFetchDate gets set without an sync, so you're probably safe skipping the sync here also. If someone is setting up a fetcher from multiple threads, or starting it while setting it up from multiple threads, they already are in a risky space.

Reply:
done